tools/tests/com.liferay.ide.xml.search.ui.tests/src/com/liferay/ide/xml/search/ui/tests/LiferayPortletXmlTests.java

```diff
@@ -15,24 +15,37 @@
 
 package com.liferay.ide.xml.search.ui.tests;
 
+import static com.liferay.ide.xml.search.ui.tests.XmlSearchTestsUtils.buildAndValidate;
 import static com.liferay.ide.xml.search.ui.tests.XmlSearchTestsUtils.deleteOtherProjects;
 import static com.liferay.ide.xml.search.ui.tests.XmlSearchTestsUtils.getTextHoverForElement;
+import static com.liferay.ide.xml.search.ui.tests.XmlSearchTestsUtils.setElementContent;
```

Comment:
Also you can import the importorder settings file under the same resources folder to get the import ordering right.  Be sure and run "organize imports" for each file that is changed.  You can get eclipse to do this automatically in the "save actions" preferences. However, don't have it format all lines or even changed lines on save, just do formatting manualy.